src/applications/edu-benefits/0994/components/OMBInfo.jsx

```diff
@@ -0,0 +1,111 @@
+import PropTypes from 'prop-types';
```

Comment:
Is this component different than https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/ombinfo/?

Reply:
yes, 
- “respondent burden” removed “(Office of Management and
     Budget) ”
- “privacy act notice” was larger change
additionally see https://dsva.slack.com/archives/CG6N59X40/p1556311972017800